pallets/farming/src/lib.rs

```diff
@@ -377,6 +379,23 @@ pub mod pallet {
     pub enum Error<T> {
         /// Increment account reference error.
         IncRefError,
+        /// Invalid New xor Min amount for Liquidity Provider Bonus Reward.
+        InvalidNewLpMinXorForBonusRewardError,
+    }
+
+    #[pallet::event]
+    #[pallet::generate_deposit(pub(super) fn deposit_event)]
+    pub enum Event<T: Config> {
+        /// When Minimum XOR amount for Liquidity Provider Bonus Reward is updated
+        LpMinXorForBonusRewardUpdated {
+            new_lp_min_xor_for_bonus_reward: Balance,
+            old_lp_min_xor_for_bonus_reward: Balance,
+        },
+    }
+
+    #[pallet::type_value]
+    pub fn DefaultLpMinXorForBonusReward<T: Config>() -> Balance {
+        balance!(1)
```

Comment:
I think it's better to set default value to one mentioned in issue to apply changes right after the next runtime upgrade